template/guestbook_message_template.tpl.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * @file
+ * Default theme implementation for guestbook messages.
+ *
+ * Available variables:
+ * - $variables['body']: an array containing the following information:
+ *   - 'id' - message id;
+ *   - 'uid' - user id;
+ *   - 'sid' - session id;
+ *   - 'date' - unix timestamp;
+ *   - 'message' - message text;
+ *   - 'user_name' - author of message;
+ *   - 'formatted_date' - formatted unix timestamp.
+ *
+ * @see theme_pager()
+ */
+?>
+<?php  global $user; ?>
+
+<div>
+  <?php foreach ($variables['body'] as $value): ?>
+    <div class = "guestbook-message">
+      <div class = "guestbook-user-date">
+        <em>
+          <?php print $value['user_name']; ?>
+          <?php print $value['formatted_date']; ?>
+        </em>
+      </div>
+
+      <div class = "guestbook-message-text">
+        <b>
+          <?php print $value['message']; ?>
+        </b>
+      </div>
+
+      <?php // If user is logged and current session id = stored id ?>
+      <?php // in database then generate html markup for ?>
+      <?php // edit button and delete button. ?>
+      <?php if ($user->uid): ?>
```

Comment:
@PavelSPN If you see the 2 conditions which one is following by another you can combine them
`if ($user->uid && $value['sid'] == $user->sid)`